src/audio/volume/volume.c

```diff
@@ -44,32 +44,27 @@
 /**
  * \brief Synchronize host mmap() volume with real value.
  * \param[in,out] cd Volume component private data.
- * \param[in] chan Channel number.
+ * \param[in] num_channels Update channels 0 to num_channels -1.
  */
-static void vol_sync_host(struct comp_data *cd, uint32_t chan)
+static void vol_sync_host(struct comp_data *cd, int num_channels)
```

Comment:
the original type `uint32_t` was unsigned. That had an advantage of the `chan < SOF_IPC_MAX_CHANNELS` check verifying all values, now with `int` you would potentially miss negative values. Better use `unsigned int`

Reply:
Fixed now.